src/storage/limits.ts

```diff
@@ -16,6 +16,21 @@ export async function getFileSizeLimit(tenantId: string): Promise<number> {
   return fileSizeLimit
 }
 
+/**
+ * Determines if the image transformation feature should be enabled.
+ * Support multi tenancy
```

Comment:
'Support multi-tenancy' is a little confusing; almost sounds like a todo. 

nits: 
- consistent hyphenation for multi-tenancy